### PR TITLE
[release-v1.14]: update rekt

### PR DESCRIPTION
**Pull request by Cali0707**
Update reconciler-test to include the fixes in https://github.com/knative-extensions/reconciler-test/pull/713/files

### CODE REVIEW

**Comment by creydr** [comment]
Same as https://github.com/openshift-knative/eventing-kafka-broker/pull/1076#issuecomment-2093169794.
Maybe you can also add a link to https://github.com/knative-extensions/reconciler-test/pull/713/files in the description, so we have a bit history

File: openshift/release/artifacts/eventing-kafka-controller.yaml
Comment (creydr) [inline openshift/release/artifacts/eventing-kafka-controller.yaml:1]: @Cali0707: where do these changes come from? Are they intended? (or just from `make generate-release`)?

Comment (Cali0707) [reply openshift/release/artifacts/eventing-kafka-controller.yaml:1]: Yeah just `make generate-release`, not sure why though. Maybe @pierDipi has an idea?

Comment (creydr) [reply openshift/release/artifacts/eventing-kafka-controller.yaml:1]: :facepalm: sorry for the noise...

**Pull request merged**